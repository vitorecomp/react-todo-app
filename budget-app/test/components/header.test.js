import React from 'react'
import { shallow } from 'enzyme'

import ReactShallowRenderer from 'react-test-renderer/shallow'


import Header from '../../src/components/header'



test('should render header conrrectly', () => {
	const wrapper = shallow(<Header />)
	expect(wrapper.find('h1').text()).toBe('Expensify')
	expect(wrapper).toMatchSnapshot();

	const renderer = new ReactShallowRenderer()
	renderer.render(<Header />)
	expect(renderer.getRenderOutput()).toMatchSnapshot();
})
