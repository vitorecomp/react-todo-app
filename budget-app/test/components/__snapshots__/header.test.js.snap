// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header conrrectly 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/create"
  >
    Create Expense
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/help"
  >
    Help Expense
  </NavLink>
</header>
`;

exports[`should render header conrrectly 2`] = `
<header>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/create"
  >
    Create Expense
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/help"
  >
    Help Expense
  </NavLink>
</header>
`;
